pkg/bosh/bpmconverter/container_factory.go

```diff
@@ -151,7 +151,7 @@ func createWaitContainer(requiredService *string) []corev1.Container {
 		return nil
 	}
 	return []corev1.Container{{
-		Name:    "wait-for",
+		Name:    fmt.Sprintf("wait-for %s", *requiredService),
```

Comment:
You are missing a dash here (assuming `requiredService` is already a valid container name).

```suggestion
		Name:    fmt.Sprintf("wait-for-%s", *requiredService),
```